.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,13 +3,15 @@ Thank you very much for contributing! Please fill out the following
 questions to make it easier for us to review your changes.
 -->
 
-# The purpose of this change
+# Change Title
```

Comment:
If it's possible to tell the validator to ignore this file or even better the entire `.github/` directory, I think we should do that. If that's not an option, I think we should s/Change Title/Pull Request/.